app/components/search-header.js

```diff
@@ -0,0 +1,4 @@
+import Ember from 'ember';
```

Comment:
Again this was to emulate ember-share, but I agree that there isn't really much need for this